views/js/helper/breadcrumbs.js

```diff
@@ -14,17 +14,17 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
  * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
  */
-(function(document) {
-    'use strict';
-
-    var amdLoader = document.getElementById('amd-loader');
-    var appConfig = amdLoader.getAttribute('data-config');
-    var appUrl = amdLoader.getAttribute('data-controller');
-    var appPage = amdLoader.getAttribute('data-action');
-    require([appConfig], function() {
-         require(['jquery', 'ui', appUrl + appPage + '.js'], function($, ui) {
-            ui.startEventComponents($('.content-wrap'));
+define(['ui/breadcrumbs'], function(breadcrumbs){
```

Comment:
ok for moving it to js/component/ however I wonder if it is really worth adding its own test since it is so small. As you can see, it just wrap the call to breadcrumbs to DRY.